### ISSUE TITLE
Teacher and Care Team can write diary entries on a Service User record.

### ISSUE THREAD

**Issue by MollieCourtenay** [healthlocker/oxleas-adhd#184]
Teachers and Care Team members can make diary entries on a Service User record.
Are these diary entries visible to the service user? / Carer?
Why is this useful over messaging?
How will this be used by clinicians and teachers?

**Comment by katbow** [comment]
Does this mean editing/adding a diary entry for a service user? So if a service user went to their diaries in tracking, would it show notes made by the teacher there?

Would this be accessed through details & contacts section?

**Comment by katyedwards** [comment]
Yes to SU view - they'll see it on their tracking overview along with entries they have made.

As for where teacher will access it - not sure, but I guess details and contacts makes sense - maybe change the name to Details and Diary or something.

**Comment by MollieCourtenay** [comment]
<img width="407" alt="screen shot 2017-09-08 at 18 29 22" src="https://user-images.githubusercontent.com/25012688/30223655-a9d5dc94-94c3-11e7-82fd-cb916e7745c0.png">

**Comment by MollieCourtenay** [comment]
(New icon needed - different coloured icon for tracking overview when a diary entry has been saved by a teacher or clinician)
